charts/thoras/templates/api-server-v2/service.yaml

```diff
@@ -10,7 +10,8 @@ metadata:
 {{- end }}
 spec:
   ports:
-  - port: {{ .Values.thorasApiServerV2.port }}
+  - name: thoras-api-server-v2
```

Comment:
Not necessarily a blocker but my understanding of the convention of port names on a service object is to describe the protocol more than application-specific things. Like name would be `https` if the port is 443 or `http` for `8080`, `grpc`, etc